app/build.gradle

```diff
@@ -18,11 +18,16 @@ android {
     }
 }
 
+repositories {
+    google()
+}
+
 dependencies {
+    compile 'com.android.support:design:27.1.1'
```

Comment:
noooooo!!!!!     I though we were gonna merge it today... :S    Please change compile to implementation... in my understanding, "compile" is deprecated now